docs/tsg_pvs.md

```diff
@@ -72,49 +82,58 @@ This issue is due to failed application operations in the container. Typically t
 
-  6. If the device is not logged in again, ensure that the network issues/failed nodes/failed replicas are resolved and device is discovered and session is re-established. This can be achieved using the commands `iscsiadm -m discovery -t st -p <ctrl svc IP>:3260` and `iscsiadm -m node -T <iqn> -l` respectively
+  6. If the device is not logged in again, ensure that the network issues/failed nodes/failed replicas are resolved, device is discovered, and session is re-established. This can be achieved using the commands `iscsiadm -m discovery -t st -p <ctrl svc IP>:3260` and `iscsiadm -m node -T <iqn> -l` respectively.
 
-  7. Identify the new SCSI device name corresponding to the iSCSI session (the device name may or may not be the same as before)
+  7. Identify the new SCSI device name corresponding to the iSCSI session (the device name may or may not be the same as before).
 
-  8. Re-mount the new disk into the mountpoint mentioned earlier using the  `mount -o rw,relatime,data=ordered /dev/sd<> <mountpoint>` command. If the remount fails due to inconsistencies on the device (unclean filesystem), perform a filesyscheck `fsck -y /dev/sd<>`.
+  8. Re-mount the new disk into the mountpoint mentioned earlier using the  `mount -o rw,relatime,data=ordered /dev/sd<> <mountpoint>` command. If the re-mount fails due to inconsistencies on the device (unclean filesystem), perform a filesyscheck `fsck -y /dev/sd<>`.
 
-  9. Ensure that the application uses the new mount by forcing it to restart on the same node. This can be done using `docker stop <id>` of the application container on the node. Kubernetes will automatically restart the pod to ensure the "desirable" state.
+  9. Ensure that the application uses the newly mounted disk by forcing it to restart on the same node. Use the command  `docker stop <id>`  of the application container on the node. Kubernetes will automatically restart the pod to ensure the "desirable" state.
 
      While this step may not be necessary most times (as the application is already undergoing periodic restarts as part of the CrashLoop cycle), it can be performed if the application pod's next restart is scheduled with an exponential back-off delay.
 
-**Note:** In environments where the kubelet runs in a container (RancherOS, CoreOS, Containerized OpenShift deployment), you may not need perform the iSCSI re-login and remount steps explicitly. Instead, it is replaced by the kubelet container restart on the node.
+**Note:**  Perform the following steps as part of the recovery procedure for a Volume-Read only issue.
```

Comment:
**Note:**  In environments where the kubelet runs in a container, perform the following steps as part of the recovery procedure for a Volume-Read only issue.